pubspec.yaml

```diff
@@ -18,15 +18,15 @@ environment:
 
 dependencies:
   build: ^0.12.7
+  build_config: ^0.3.1
   built_value: ^5.1.0
   matcher: ^0.12.0+1
   quiver: ^0.29.0
-  source_gen: ^0.8.0
+  source_gen: ^0.9.0
   webdriver: ^2.0.0-beta+1
```

Comment:
`webdriver` version will need to be updated too -- see https://github.com/google/webdriver.dart/issues/205#issuecomment-410734933.

Reply:
Pending - https://github.com/google/webdriver.dart/pull/206